src/backend/distributed/replication/multi_logical_replication.c

```diff
@@ -462,34 +720,33 @@ PrepareReplicationSubscriptionList(List *shardList)
 
 
 /*
- * TableOwnerIds returns a bitmapset containing all the owners of the tables
- * that the given shards belong to.
+ * CreateReplicaIdentities creates replica identities for all the shards that
+ * are part of the given subscriptions.
  */
-static Bitmapset *
-TableOwnerIds(List *shardList)
+void
+CreateReplicaIdentities(List *subscriptionInfoList)
```

Comment:
Would be good to take a list of connections rather than subscriptionInfoList. It might act as a helper?

Reply:
I think the renaming of subscriptionInfoList to logicalRepTargetList makes this a lot clearer.